docs/HARDWARE-VERIFICATION.md

```diff
@@ -87,12 +87,24 @@ the read performance of the available storage devices using
 ## Getting the results of the verification process
 
 The verification image prints the results of tests on the screen and stores
-them when possible (e.g., when we are using the raw version) in the
-**inventory partition** of the boot media. We can get the verification process
-results by running the command ```tools/extract_verification_info.sh <verification.raw>```
+them when possible (e.g., when we are using a USB or a raw file) in the
+**inventory partition** of the boot media. We can extract the logs of the
+verification process results by running the command
+```tools/extract-verification-info.sh <USB_device_name|verification_img>```
 on the `verification.raw` image file or on the USB stick. To achieve that, we must
-mount the USB stick or copy the image file to a PC and execute the script. The
+mount the USB stick or copy the image file to our PC and execute the script. The
 script copies, among other things, the logs of the tests, details about the
 hardware, etc. A file named ```summary.log``` contains a summary of the
 verification process results to help the user quickly understand potential
 problems.
+
+## Publishing the results of the verification processes
+
+Apart from extracting the logs of the verification process, we can upload
+them in a dedicated web application (currently hosted on `https://centaur.med.uoc.gr:8999`).
```

Comment:
I have asked @yash-zededa to create a new repo for the web app's source code and am waiting for it. In the meantime, I will put some instructions in its Readme that describe how to deploy it.

Reply:
But even with a place to upload it isn't clear that we want the world to publish their photo album on publicly available web server which does not require authentication. 
So I think the documentation needs to say
 Assuming a https server is running on www.example.com:8999, one can publish the results by running the script like this:

Thus using the reserved www.example.com name instead of a specific one.

Reply:
Done.